submissions/ustymchyk/memory-pair-game/index.js

```diff
@@ -0,0 +1,94 @@
+class Game {
+    constructor(selector) {
+        this.gameContainer = document.querySelector(selector);
+        this.rightAnswers = 0;
+        this.openCards = [];
+    }
+
+    startGame() {
+        this.gameContainer.innerHTML = this.getCards();
+        this.addListeners();
+    }
+
+    addListeners() {
```

Comment:
Use [event delegation](https://javascript.info/event-delegation), please. It'll help you to increase the performance of your app.